### PR TITLE
Folder added : Assignment 8

### PR DESCRIPTION
**Pull request by sourabh-kanwade**

### CODE REVIEW

**Review by somesh6797 — APPROVED** [review]

**Review by shivling123 — APPROVED** [review]

**Review by kalyani1942 — APPROVED** [review]

**Review by SnehaMakade — APPROVED** [review]

File: Sourabh_Kanwade/html/Assignment 8/question1.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                padding: 2rem;
+                width: 70%;
+                background-color: aqua;
+                border: solid 2px orange;
+            }
+            .img-container {
+                display: flex;
+                justify-content: space-between;
+            }
+            .img-container > img {
```
Comment (shivling123) [inline Sourabh_Kanwade/html/Assignment 8/question1.html:19]: remove >

File: Sourabh_Kanwade/html/Assignment 8/question1.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                width: 200px;
+                height: 200px;
+            }
+            .img-container > img, .img-container > p {
```
Comment (shivling123) [inline Sourabh_Kanwade/html/Assignment 8/question1.html:23]: check syntax

File: Sourabh_Kanwade/html/Assignment 8/question1.html (hunk 1 above)
Comment (Phantom-Code) [reply Sourabh_Kanwade/html/Assignment 8/question1.html:19]: `>` for selecting child elements
I wanted to select child `` element from `.img-container`

File: Sourabh_Kanwade/html/Assignment 8/question1.html (hunk 2 above)
Comment (Phantom-Code) [reply Sourabh_Kanwade/html/Assignment 8/question1.html:23]: what should I change?

**Review by shivling123 — APPROVED** [review]

**Pull request merged**